tests/test_create.py

```diff
@@ -2612,11 +2632,11 @@ def test_repodata_v2_base_url(
     monkeypatch: MonkeyPatch,
     request: FixtureRequest,
 ):
-    if context.solver == "libmamba":
+    if context.solver == "libmamba" and version("libmambapy").startswith(("0.", "1.")):
```

Comment:
We usually use the full version parser in cases like this

Reply:
This won't last more than the current release cycle. It's mostly to have green CI for the merge, so I didn't put too much effort. But sure, I'll use `VersionOrder`.